plugins/kafka/kafka.cpp

```diff
@@ -475,7 +475,14 @@ namespace KafkaPlugin
     {
         if (message.err() != RdKafka::ERR_NO_ERROR)
         {
-            DBGLOG("Kafka: Error publishing message: %d (%s); message: '%s'", message.err(), message.errstr().c_str(), static_cast<char*>(message.payload()));
```

Comment:
Use of %.*s and passing the length in is generally preferable to making an extra copy just to null terminate, I would have thought